libs/platform/src/lib/components/dynamic-page/dynamic-page-footer/dynamic-page-footer.component.ts

```diff
@@ -0,0 +1,20 @@
+import { Component, OnInit, ChangeDetectionStrategy, forwardRef } from '@angular/core';
+
+import { DYNAMIC_PAGE_CHILD_TOKEN } from '../constants';
+
+@Component({
+    selector: 'fdp-dynamic-page-footer',
+    template: '<div style="position:relative"><ng-content></ng-content></div>',
+    changeDetection: ChangeDetectionStrategy.OnPush,
+    providers: [
+        {
+            provide: DYNAMIC_PAGE_CHILD_TOKEN,
+            useExisting: forwardRef(() => DynamicPageFooterComponent)
+        }
+    ]
+})
+export class DynamicPageFooterComponent implements OnInit {
+    constructor() {}
```

Comment:
empty methods we can remove. The same is applicable for other places as well.

Reply:
removed